jobs/build/aws-ami/Jenkinsfile

```diff
@@ -81,30 +85,30 @@ openshift_cloudprovider_aws_access_key: ${env.AWS_ACCESS_KEY_ID}
 openshift_aws_base_ami_name: ${BASE_AMI_NAME}
 openshift_additional_repos: [{'name': 'openshift-repo', 'id': 'openshift-repo',  'baseurl': '${env.YUM_BASE_URL}', 'enabled': 'yes', 'gpgcheck': 0, 'sslverify': 'no', 'sslclientcert': '/var/lib/yum/client-cert.pem', 'sslclientkey': '/var/lib/yum/client-key.pem', 'gpgkey': 'https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted'},{'sslverify': False, 'name': 'fastdata', 'sslclientkey': '/var/lib/yum/client-key.pem', 'enabled': True, 'gpgkey': 'https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted', 'sslclientcert': '/var/lib/yum/client-cert.pem', 'baseurl': 'https://mirror.ops.rhcloud.com/enterprise/rhel/rhel-7-fast-datapath-rpms/', 'file': 'fastdata-ovs', 'gpgcheck': False, 'description': 'Fastdata provides the official builds of OVS OpenShift supports'}]
-"""
-
-                    sshagent(["${AWS_SSH_KEY_USER}"]) {
-                        // need to load aws credentials
-                        withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: 'ami-build-creds']]) {
-                        sh("cat ${WORKSPACE}/provisioning_vars.yml")
-                        sh(script: "ANSIBLE_HOST_KEY_CHECKING=False AWS_ACCESS_KEY_ID=${env.AWS_ACCESS_KEY_ID} AWS_SECRET_ACCESS_KEY=${env.AWS_SECRET_ACCESS_KEY} ansible-playbook ${WORKSPACE}/openshift-ansible/playbooks/aws/openshift-cluster/build_ami.yml -e @${WORKSPACE}/provisioning_vars.yml -vvv")
+""")
+            }
+            sh 'cat provisioning_vars.yml'
+            withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: 'ami-build-creds']]) {
+                withEnv(['ANSIBLE_HOST_KEY_CHECKING=False']) {
+                    sshagent([AWS_SSH_KEY_USER]) {
+                        buildlib.with_virtualenv('env') {
+                            sh 'ansible-playbook openshift-ansible/playbooks/aws/openshift-cluster/build_ami.yml -e @provisioning_vars.yml -vvv'
                         }
                     }
                 }
             }
-        } catch ( err ) {
-            // Replace flow control with: https://jenkins.io/blog/2016/12/19/declarative-pipeline-beta/ when available
-            mail(to: "${MAIL_LIST_FAILURE}",
-                    from: "kwoodson@redhat.com",
-                    subject: "Error building aws ami",
-                    body: """Encoutered an error while running build_ami.yml: ${err}
+        }
+    } catch ( err ) {
+        // Replace flow control with: https://jenkins.io/blog/2016/12/19/declarative-pipeline-beta/ when available
+        mail(to: "${MAIL_LIST_FAILURE}",
+                from: "kwoodson@redhat.com",
```

Comment:
Would love to change this from address :)

Reply:
Somehow I read that as 'to', not 'from'.  Changed it to the address we use on
other jobs.